spec/index.bs

```diff
@@ -605,7 +603,7 @@ interface GPUProgrammablePassEncoder : GPUObjectBase {
     void insertDebugMarker(DOMString markerLabel);
 };
 
-interface GPURenderPassEncoder : GPUProgrammablePassEncoder {
+interface GPURenderEncoderBase : GPUProgrammablePassEncoder {
```

Comment:
one other limitation is a bit concerning: inability to change/set descriptor heaps in D3D12 Bundles. Do you have a plan on how the implementation is going to achieve this?

Reply:
I don't have a plan. Implementations could have a specific heap for the bundle and set it up before executing it. I don't see other solution that wouldn't heavily constrain what can be done in bundles and like @kainino0x I hope that bundles are as unconstrained as possible.

Reply:
What about binding inheritance for bundles? If we have inheritance (say, as a run-time flag), we don't know which bind groups are going to be used when we are recording the bundle, so we can't put everything in the same descriptor heap ahead of time.

Reply:
We should probably discuss inheritance in the group, but what do you think of declaring what's inherited from the renderpass in the `GPURenderBundleEncoderDescriptor` so that at creation developers have to declare what will be inherited?

Reply:
I think having this flag would be good, even though the particular implementation details are unclear.